code/client/makepkginfo

```diff
@@ -385,6 +385,15 @@ def check_mode(option, opt, value, parser):
                                                             (opt, mode))
     setattr(parser.values, option.dest, value)
 
+def has_valid_install_critieria(catinfo):
```

Comment:
if installer_type == 'profile' and PayloadIdentifier is absent or None the pkginfo is not a valid pkginfo. I don't think we can get to this state, and anyway, it's outside of the intention of this function.